.copier-answers.yml

```diff
@@ -1,13 +1,14 @@
 # Do NOT update manually; changes here will be overwritten by Copier
-_commit: v1.11.1
+_commit: v1.14.2
 _src_path: https://github.com/OCA/oca-addons-repo-template.git
 ci: GitHub
 dependency_installation_mode: PIP
 generate_requirements_txt: true
 github_check_license: true
-github_enable_codecov: true
+github_ci_extra_env: {}
+github_enable_codecov: false
 github_enable_makepot: false
-github_enable_stale_action: true
+github_enable_stale_action: false
```

Comment:
@AungKoKoLin1997 I prefer to keep it true.  It's a minor thing so we don't need to revert it this time.